### ISSUE TITLE
` in Character Creation Brings Up Error

### ISSUE THREAD

**Issue by colearkenach** [RolePlayGateway/roleplaygateway#58]
https://cdn.discordapp.com/attachments/335217640660533269/335245028819730432/JPEG_20170713_212349.jpg

The following error is brought up upon attempting character creation according to the user Draumr_Drac. This occurred when attempting to make a character for the MV.

**Comment by oleande-r** [comment]
I was able to somewhat recreate the error, as I have experienced this in the past.
![](https://i.imgur.com/iSPfFGt.png)

As far as I know, the error occurs whenever you enter in a text symbol that the site doesn't register (idk if that's the correct terminology). For example, if I used this symbol (🐉) I would get the error since the site doesn't register the symbol, however if I used this symbol (☏) the error wouldn't occur because it's a symbol that the site recognizes.

The string of numbers & letters ( \xF0\x9F\x85\xB6\xF0\x9F ) changes depending on which symbol you use.

**Comment by colearkenach** [comment]
Makes sense. I'll be sure to pass the information along to the user who had this problem.

**Comment by martindale** [comment]
Aha, yes.  This looks like a character encoding problem.  Maybe we need to fix the configuration of the database, or maybe PHP itself.  There's documentation for this somewhere.

**Comment by lostamongtrees** [comment]
> Aha, yes. This looks like a character encoding problem. Maybe we need to fix the configuration of the database, or maybe PHP itself. There's documentation for this somewhere.

see:  #34  , #35 in regards to PHP 
has this been resolved?

**Comment by lostamongtrees** [comment]
edit: realized, as i had copypastad the name, it was not an apostraphe but a ....the other apostraphe.  So that might be the cause of literally it all.

Received this error when creating an anon character with a ` ` ` in the name

![image](https://user-images.githubusercontent.com/52435234/62835040-c1084c80-bc21-11e9-90d8-a0e8ad6f2173.png)

ther was a php on refresh but i wasnt fast enough to screenshot, will try to replicate

interestingly enough, after refresh was brought here

![image](https://user-images.githubusercontent.com/52435234/62835052-d8473a00-bc21-11e9-83f2-6c9df99d0da3.png)

`back` brought me back to character creation with all data saved.
Character exists, but is broken.

https://www.roleplaygateway.com/universes/the-multiverse/characters/ni%e2%80%99thorne
![image](https://user-images.githubusercontent.com/52435234/62835068-02006100-bc22-11e9-82dc-f87a9cf1c572.png)

when i go to edit the character

![image](https://user-images.githubusercontent.com/52435234/62835079-23f9e380-bc22-11e9-837a-f5e1cb681487.png)

**Comment by lostamongtrees** [comment]
Renaned as we seem to have isolated it to the special character of `
